Makefile.toml

```diff
@@ -466,7 +466,10 @@ os = os_family
 if eq ${os} windows
   exec --fail-on-error magick convert -font arial -pointsize 14 -fill "#FFA400" -gravity south -annotate +0+5 ${version} ./resources/images/splash.jpg ./resources/images/splash-version.jpg
 else
-  exec --fail-on-error convert -font helvetica -pointsize 14 -fill "#FFA400" -gravity south -annotate +0+5 ${version} ./resources/images/splash.jpg ./resources/images/splash-version.jpg
+  output = exec convert -font helvetica -pointsize 14 -fill "#FFA400" -gravity south -annotate +0+5 ${version} ./resources/images/splash.jpg ./resources/images/splash-version.jpg
```

Comment:
oh yeah why did we need to do this btw? *just noticed we use helvetica on non windows

Reply:
Helvetica was not available on my Ubuntu 22.04 system, so this was the best workaround available.